public/js/graph/react_graph.js

```diff
@@ -438,6 +440,34 @@ var Graph = React.createClass({
         });
     },
 
+    onButtonPress: function(zoomOrPanFunction, direction) {
+        this.setState({mouseDown: setInterval(() => zoomOrPanFunction(direction), 100)});
```

Comment:
Thanks for the feedback and those resources. throttle seems interesting and could probably work here. 

I will continue to work on ways to improve the current implementation.